docs/registries.conf.5.md

```diff
@@ -0,0 +1,26 @@
+% registries.conf(5) System-wide registry configuration file
+% Brent Baude
+% Aug 2017
+
+# NAME
+registry.conf - Syntax of System Registry Configuration File
+
+# DESCRIPTION
+The REGISTRIES configuration file is a system-wide configuration file for container image
+registries. The file format is TOML.
+
+# FORMAT
+The [TOML_format][toml] is used to build simple list format for registries under two
+categories: `registries` and `insecure_registries`. You can list multiple registries using
```

Comment:
image_registries, registries and insecure_registries would work for me.

I think I'm still a bit fuzzy about the difference.  Could dockerhub registry without authentication and then a private registry inside the company with authentication both be listed under "additonal/registries"?  I was thinking the dockerhub would be in the "insecure_registries" and that's where I was going with my naming.